gfauto/gfauto/amber_converter.py

```diff
@@ -445,6 +455,9 @@ def to_shader_job(self) -> ShaderJob:
             must_exist=True,
         )
 
+        # Figure out if we want to draw rectangle or grid
```

Comment:
Figure out if we want to draw rectangle or grid
->
Figure out if we want to draw a rectangle or grid. (Add "a"; end in full stop.)